web-console/src/utils/capabilities.ts

```diff
@@ -55,7 +56,7 @@ export class Capabilities {
     // Check SQL endpoint
     try {
       await Api.instance.post(
-        '/druid/v2/sql',
+        '/druid/v2/sql?capabilities',
```

Comment:
is the `?capabilities` query parameter just for fun?

Reply:
it is mostly for being able to identify these bad boys in the Network tab:

<img width="467" alt="image" src="https://user-images.githubusercontent.com/177816/167033255-0e033b4f-d97b-420b-9a2b-2acdc11b3a7c.png">